2019-11-13 18:58:10,342 : INFO : Machine : Mission Pinball Framework Core Engine v0.53.0-dev.67
2019-11-13 18:58:10,343 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': False, 'configfile': ['config.yaml'], 'mpfconfigfile': 'c:\\users\\schol\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\mpf\\mpfconfig.yaml', 'force_assets_load': False, 'jsonlogging': False, 'logfile': 'logs\\2019-11-13-18-58-10-mpf-DESKTOP-FPTMTCL.log', 'pause': False, 'production': False, 'text_ui': True, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': None, 'syslog_address': None, 'mc_file_name': None, 'no_sound': False}
2019-11-13 18:58:10,343 : INFO : Machine : MPF path: c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf
2019-11-13 18:58:10,343 : INFO : Machine : Machine path: D:\mpf\wows\wows
2019-11-13 18:58:10,343 : INFO : Machine : Platform: win32
2019-11-13 18:58:10,343 : INFO : Machine : Python executable location: c:\users\schol\appdata\local\programs\python\python36\python.exe
2019-11-13 18:58:10,343 : INFO : Machine : Python version: 3.6.4 (64-bit)
2019-11-13 18:58:10,343 : INFO : Machine : Machine config file #1: config.yaml
2019-11-13 18:58:10,344 : WARNING : ConfigProcessor : Config file in cache changed: D:\mpf\wows\wows\config\config.yaml
2019-11-13 18:58:10,344 : INFO : ConfigProcessor : Loading config from file c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\mpfconfig.yaml.
2019-11-13 18:58:10,420 : INFO : ConfigProcessor : Loading config: c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\mpfconfig.yaml
2019-11-13 18:58:10,427 : INFO : ConfigProcessor : Loading config from file D:\mpf\wows\wows\config\config.yaml.
2019-11-13 18:58:10,447 : INFO : ConfigProcessor : Loading config: D:\mpf\wows\wows\config\config.yaml
2019-11-13 18:58:10,450 : ERROR : root : Config File Error in ConfigProcessor: Found a "enable_events:" section in config file D:\mpf\wows\wows\config\config.yaml, but that section name is unknown. Did you mean "ball_devices:" instead?. Context: D:\mpf\wows\wows\config\config.yaml Error Code: CFE-ConfigProcessor-3 (https://docs.missionpinball.org/en/dev/logs/CFE-ConfigProcessor-3.html)
Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 146, in _check_sections
    if config_type not in config_spec[k]['__valid_in__']:
KeyError: 'enable_events'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\commands\game.py", line 226, in __init__
    self.machine = MachineController(mpf_path, machine_path, vars(self.args))
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 199, in __init__
    self._load_config()
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 431, in _load_config
    store_to_cache=self.options['create_config_cache'])
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 113, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 178, in _load_config_file_and_return_loaded_files
    self._check_sections(config, config_type, filename, ignore_unknown_sections)
  File "c:\users\schol\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 159, in _check_sections
    filename)
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigProcessor: Found a "enable_events:" section in config file D:\mpf\wows\wows\config\config.yaml, but that section name is unknown. Did you mean "ball_devices:" instead?. Context: D:\mpf\wows\wows\config\config.yaml Error Code: CFE-ConfigProcessor-3 (https://docs.missionpinball.org/en/dev/logs/CFE-ConfigProcessor-3.html)
